### ISSUE TITLE
Incorrect use of term 'saturated'

### ISSUE THREAD

**Issue by cappie** [RealFaviconGenerator/realfavicongenerator#55]
"Use a **saturated** version of the favicon. Works well with pictures with significant transparent regions."

This should be:

"Use a **desaturated** version of the favicon. Works well with pictures with significant transparent regions."

Saturated images contain MORE colors.. desaturated images contain LESS color.

But since what you're actually doing is creating a white silhouette of the image, thus completely destroying, it might even be more relevant just using this:

"Use a **white silhouette** version of the favicon. Works well with pictures with significant transparent regions."

**Comment by phbernard** [comment]
Thank for the feedback! I speak English but I'm not fluent. There are many cases where I'm not so sure of the words I use. This "saturated" sentence was one of them. In addition, I'm a developer, not a designer and all these graphics terms are not that clear to me, whatever the language. Double dilemma :)

I picked your last recommendation, "white silhouette", which definitely makes sense. Fixed and deployed a minute ago.
